app/javascript/packages/document-capture/components/in-person-full-address-search.tsx

```diff
@@ -147,10 +178,20 @@ function FullAddressSearch({
       input(target.value);
     };
 
+  const inputChangeHandlerForZipCode =
+    <T extends HTMLElement & { value: string }>(input) =>
+    (event: React.ChangeEvent<T>) => {
+      const { target } = event;
+      const scrubbedZip = target.value.replace(/[^0-9]/g, '');
+      input(scrubbedZip);
+    };
```

Comment:
The remaining code will still cause cursor jumping, so we should remove the regex replace.

Reply:
I am not find a way to only allow digits in the input with a max length of 5. I have combed the docs several times. Have any good ideas?

Option 1: type=number with min=11111 and max=99999. (maxLength and minLength do not work with type=number)
- A user can type in more than 5 digits. -
- A non-digit input would be ignored. +

Option 2: type=tel or type=text with maxLength=5 and minLength=5. 
- A user can type in letters. -
- Input will be cut off at 5. +

Option 3: use replace
- length is only 5 +
- input will only be digits +
- cursor jumps -


In theory, if we work on the zip code component in the next sprint- this solution will only be in production for a few weeks- and only when the `in_person_full_address_entry_enabled` is enabled. Willing and able to try any other solutions and will keep digging. I listed options and pros + / cons - for each solution.

Reply:
Option 2 with `type=text` works, the user will still see an error if they try to enter digits because of the pattern validation.

Reply:
@NavaTim  Resolved with commit 03681082431676faff7bb4e5db53aab1636db224 - ready for final review